### ISSUE TITLE
Root folders not being pulled (I have more than the usual number of root folders)

### ISSUE THREAD

**Issue by BEisem** [Radarr/Radarr#4905]
**Describe the bug**
I have a specific setup that uses more than the usual number of root folders. I organize alphabetically, and I have two sets (one local, one on Google Drive). Plus a few others. So the total number of root folders is 57.

**To Reproduce**
1. Create 57 root folders
2. Go to Movies / Import or Settings / Media Management (anywhere that lists the root folders).
3. View the GUI where the root folders should be listed.
4. Radarr works on it for several minutes, and eventually it displays "Unable to load root folders."

**Expected behavior**
The root folders should be listed.

**Screenshots**

**Platform Information (please complete the following information):**
 - OS: Docker
 - Mono Version: [e.g. Mono 5.8] (Only needed under Linux and Mac, found under System -> Status)
 - Browser and Version [e.g. chrome, safari] (Only needed for UI issues)
 - Radarr Version [e.g. 3.0.0.3456]

**Debug Logs**

today at 7:20 PM [s6-init] making user provided files available at /var/run/s6/etc...exited 0.
today at 7:20 PM [s6-init] ensuring user provided files have correct perms...exited 0.
today at 7:20 PM [fix-attrs.d] applying ownership & permissions fixes...
today at 7:20 PM [fix-attrs.d] done.
today at 7:20 PM [cont-init.d] executing container initialization scripts...
today at 7:20 PM [cont-init.d] 00-start-container: executing...
today at 7:20 PM
today at 7:20 PM ----------------------------------------------------------------------
today at 7:20 PM ENVIRONMENT
today at 7:20 PM ----------------------------------------------------------------------
today at 7:20 PM PUID=1000
today at 7:20 PM PGID=998
today at 7:20 PM UMASK=002
today at 7:20 PM TZ=America/New_York
today at 7:20 PM ARGS=
today at 7:20 PM DEBUG=yes
today at 7:20 PM ----------------------------------------------------------------------
today at 7:20 PM
today at 7:20 PM Executing usermod...
today at 7:20 PM usermod: no changes
today at 7:20 PM Applying permissions to /config
today at 7:20 PM [cont-init.d] 00-start-container: exited 0.
today at 7:20 PM [cont-init.d] done.
today at 7:20 PM [services.d] starting services
today at 7:20 PM [services.d] done.
today at 7:20 PM [Info] Bootstrap: Starting Radarr - /app/bin/Radarr.dll - Version 3.0.0.3456
today at 7:20 PM [Info] AppFolderInfo: Data directory is being overridden to [/config/app]
today at 7:20 PM [Info] Router: Application mode: Interactive
today at 7:20 PM [Info] MigrationController: *** Migrating data source=/config/app/radarr.db;cache size=-20000;datetimekind=Utc;journal mode=Wal;pooling=True;version=3 ***
today at 7:20 PM [Info] MigrationController: *** Migrating data source=/config/app/logs.db;cache size=-20000;datetimekind=Utc;journal mode=Wal;pooling=True;version=3 ***
today at 7:20 PM [Info] WebHostController: Listening on the following URLs:
today at 7:20 PM [Info] WebHostController:   http://*:7878
today at 7:20 PM [Info] RadarrBootstrapper: Starting Web Server
today at 7:20 PM [Info] CommandExecutor: Starting 2 threads for tasks.
today at 7:20 PM [Warn] Microsoft.AspNetCore.Server.Kestrel: Heartbeat took longer than "00:00:01" at "08/19/2020 23:20:41 +00:00". This could be caused by thread pool starvation.
today at 7:20 PM [Info] RssSyncService: Starting RSS Sync
today at 7:21 PM [Error] Newznab: An error occurred while processing indexer feed.
today at 7:21 PM
today at 7:21 PM [v3.0.0.3456] System.Xml.XmlException: Unexpected XML
today at 7:21 PM at NzbDrone.Core.Indexers.Newznab.NewznabCapabilitiesProvider.ParseCapabilities(HttpResponse response) in D:\a\1\s\src\NzbDrone.Core\Indexers\Newznab\NewznabCapabilitiesProvider.cs:line 85
today at 7:21 PM at NzbDrone.Core.Indexers.Newznab.NewznabCapabilitiesProvider.FetchCapabilities(NewznabSettings indexerSettings) in D:\a\1\s\src\NzbDrone.Core\Indexers\Newznab\NewznabCapabilitiesProvider.cs:line 66
today at 7:21 PM at NzbDrone.Core.Indexers.Newznab.NewznabCapabilitiesProvider.<>c__DisplayClass4_0.<GetCapabilities>b__0() in D:\a\1\s\src\NzbDrone.Core\Indexers\Newznab\NewznabCapabilitiesProvider.cs:line 34
today at 7:21 PM at NzbDrone.Common.Cache.Cached`1.Get(String key, Func`1 function, Nullable`1 lifeTime) in D:\a\1\s\src\NzbDrone.Common\Cache\Cached.cs:line 88
today at 7:21 PM at NzbDrone.Core.Indexers.Newznab.NewznabCapabilitiesProvider.GetCapabilities(NewznabSettings indexerSettings) in D:\a\1\s\src\NzbDrone.Core\Indexers\Newznab\NewznabCapabilitiesProvider.cs:line 34
today at 7:21 PM at NzbDrone.Core.Indexers.Newznab.Newznab.get_PageSize() in D:\a\1\s\src\NzbDrone.Core\Indexers\Newznab\Newznab.cs:line 23
today at 7:21 PM at NzbDrone.Core.Indexers.Newznab.Newznab.GetRequestGenerator() in D:\a\1\s\src\NzbDrone.Core\Indexers\Newznab\Newznab.cs:line 27
today at 7:21 PM at NzbDrone.Core.Indexers.HttpIndexerBase`1.FetchReleases(Func`2 pageableRequestChainSelector, Boolean isRecent) in D:\a\1\s\src\NzbDrone.Core\Indexers\HttpIndexerBase.cs:line 91
today at 7:21 PM
today at 7:21 PM [Info] DownloadDecisionMaker: Processing 855 releases
today at 7:21 PM [Warn] AcceptableSizeSpecification: [Clownface (2020)][tt4773982, 517984] has no runtime information using median movie runtime of 110 minutes.
today at 7:21 PM [Warn] AcceptableSizeSpecification: [Clownface (2020)][tt4773982, 517984] has no runtime information using median movie runtime of 110 minutes.
today at 7:21 PM [Warn] AcceptableSizeSpecification: [Clownface (2020)][tt4773982, 517984] has no runtime information using median movie runtime of 110 minutes.
today at 7:22 PM [Warn] AcceptableSizeSpecification: [Clownface (2020)][tt4773982, 517984] has no runtime information using median movie runtime of 110 minutes.
today at 7:22 PM [Warn] AcceptableSizeSpecification: [Clownface (2020)][tt4773982, 517984] has no runtime information using median movie runtime of 110 minutes.
today at 7:22 PM [Warn] AcceptableSizeSpecification: [Clownface (2020)][tt4773982, 517984] has no runtime information using median movie runtime of 110 minutes.
today at 7:22 PM [Warn] AcceptableSizeSpecification: [Clownface (2020)][tt4773982, 517984] has no runtime information using median movie runtime of 110 minutes.
today at 7:22 PM [Info] RssSyncService: RSS Sync Completed. Reports found: 855, Reports grabbed: 0
today at 7:22 PM [Info] RssSyncService: RSS Sync Completed. Reports found: 855, Reports grabbed: 0

**Comment by austinwbest** [comment]
Open the dev console, it may give better error information. I was thinking maybe the length is to much for local.storage keys but im leaning away from that. Any console errors may help shine some light on it.

**Comment by BEisem** [comment]
Here you go:

trycatch.js:131 GET https://**REDACTED**/api/v3/rootFolder 524
(anonymous)	@	trycatch.js:131
(anonymous)	@	instrument.js:220
send	@	xhr.js:151
ajax	@	ajax.js:699
se	@	createAjaxRequest.js:51
(anonymous)	@	createFetchHandler.js:14
(anonymous)	@	thunks.js:13
(anonymous)	@	index.js:8
(anonymous)	@	middleware.js:22
(anonymous)	@	createSentryMiddleware.js:54
(anonymous)	@	redux.js:477
r.componentDidMount	@	ImportMovieSelectFolderConnector.js:37
is	@	react-dom.production.min.js:212
hu	@	react-dom.production.min.js:255
t.unstable_runWithPriority	@	scheduler.production.min.js:19
Go	@	react-dom.production.min.js:122
fu	@	react-dom.production.min.js:248
Zs	@	react-dom.production.min.js:239
(anonymous)	@	react-dom.production.min.js:123
t.unstable_runWithPriority	@	scheduler.production.min.js:19
Go	@	react-dom.production.min.js:122
qo	@	react-dom.production.min.js:123
Vo	@	react-dom.production.min.js:122
eu	@	react-dom.production.min.js:240
notify	@	Subscription.js:19
t.notifyNestedSubs	@	Subscription.js:92
t.handleChangeWrapper	@	Subscription.js:97
v	@	redux.js:221
(anonymous)	@	index.js:11
(anonymous)	@	middleware.js:22
(anonymous)	@	createSentryMiddleware.js:54
dispatch	@	redux.js:638
(anonymous)	@	createFetchHandler.js:21
f	@	callbacks.js:84
fireWith	@	callbacks.js:214
f	@	deferred.js:183
h	@	deferred.js:192
r	@	helpers.js:71
setTimeout (async)		
(anonymous)	@	trycatch.js:28
(anonymous)	@	deferred.js:230
f	@	callbacks.js:84
fireWith	@	callbacks.js:214
fire	@	callbacks.js:222
f	@	callbacks.js:84
fireWith	@	callbacks.js:214
N	@	ajax.js:803
(anonymous)	@	xhr.js:99
r	@	helpers.js:71
load (async)		
s	@	object.js:38
(anonymous)	@	trycatch.js:111
(anonymous)	@	trycatch.js:109
(anonymous)	@	instrument.js:220
send	@	xhr.js:151
ajax	@	ajax.js:699
se	@	createAjaxRequest.js:51
(anonymous)	@	createFetchHandler.js:14
(anonymous)	@	thunks.js:13
(anonymous)	@	index.js:8
(anonymous)	@	middleware.js:22
(anonymous)	@	createSentryMiddleware.js:54
dispatchFetchNetImports	@	PageConnector.js:157
r.componentDidMount	@	PageConnector.js:194
is	@	react-dom.production.min.js:212
hu	@	react-dom.production.min.js:255
t.unstable_runWithPriority	@	scheduler.production.min.js:19
Go	@	react-dom.production.min.js:122
fu	@	react-dom.production.min.js:248
Zs	@	react-dom.production.min.js:239
Ks	@	react-dom.production.min.js:230
Hu	@	react-dom.production.min.js:281
(anonymous)	@	react-dom.production.min.js:284
tu	@	react-dom.production.min.js:240
Wu	@	react-dom.production.min.js:284
t.render	@	react-dom.production.min.js:290
(anonymous)	@	index.js:15
i	@	bootstrap:79
n	@	bootstrap:45
(anonymous)	@	bootstrap:152
(anonymous)	@	index.js?h=uwC1yDrtC7aJz+MyB5hEbw:1
